frontend/src/__mocks__/mockAcceleratorProfile.ts

```diff
@@ -0,0 +1,22 @@
+import { AcceleratorKind } from '~/k8sTypes';
+
+export const mockAcceleratorProfile = (): AcceleratorKind => ({
+  apiVersion: 'accelerator.k8s.io/v1',
```

Comment:
this is not the api version

Reply:
create a CR in your cluster and use that as a reference

Reply:
`dashboard.opendatahub.io/v1` ?

Reply:
``` yaml
  kind: 'AcceleratorProfile',
  apiVersion: 'dashboard.opendatahub.io/v1',
```